openedx/core/djangoapps/course_roles/rules.py

```diff
@@ -0,0 +1,31 @@
+"""
+django-rules and Bridgekeeper rules for course_roles related features
+"""
+from bridgekeeper.rules import Rule
+
+from opaque_keys.edx.keys import CourseKey
+from xblock.core import XBlock
+
+from openedx.core.djangoapps.course_roles.helpers import course_or_organization_or_instance_permission_check
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from xmodule.course_block import CourseBlock  # lint-amnesty, pylint: disable=wrong-import-order
```

Comment:
It would be nice if we could resolved the amnestied pylint warnings instead of just moving them to a new home.